aspnet/RVTR.Lodging.WebApi/Startup.cs

```diff
@@ -62,6 +62,7 @@ public void ConfigureServices(IServiceCollection services)
 
       services.AddScoped<ClientZipkinMiddleware>();
       services.AddScoped<UnitOfWork>();
+      services.AddScoped<LodgingRepo>();
```

Comment:
Don't need to add this because unit of work takes care of it

Reply:
resolved.